client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -7,17 +7,17 @@ import React from 'react';
 /**
  * Internal dependencies
  */
-import Card from 'components/card';
 import DomainMainPlaceholder from 'my-sites/upgrades/domain-management/components/domain/main-placeholder';
-import EnableDomainLockingNotice from './enable-domain-locking-notice';
-import EnablePrivacyNotice from './enable-privacy-notice';
 import Header from 'my-sites/upgrades/domain-management/components/header';
-import IcannVerificationNotice from './icann-verification-notice';
 import Main from 'components/main';
 import paths from 'my-sites/upgrades/paths';
-import PendingTransferNotice from './pending-transfer-notice';
-import RequestTransferCode from './request-transfer-code';
-import TransferProhibitedNotice from './transfer-prohibited-notice';
+import { getSelectedDomain } from 'lib/domains';
+import IcannVerification from 'my-sites/upgrades/domain-management/transfer/icann-verification';
+import Locked from 'my-sites/upgrades/domain-management/transfer/locked';
+import Unlocked from 'my-sites/upgrades/domain-management/transfer/unlocked';
+import TransferProhibited from 'my-sites/upgrades/domain-management/transfer/transfer-prohibited';
+import PendingTransfer from 'my-sites/upgrades/domain-management/transfer/pending-transfer';
+import omit from 'lodash/object/omit'
```

Comment:
Hahahahaha, I see what you did there :+1:
We can leave this as-is as a meta joke :grinning: